SU-SWS/ansible-playbooks
# SITES-482: JS Injector

# READY FOR REVIEW

# Summary
- Replace strings in JS Injector rules and files

# Needed By (Date)
- End of sprint (5/17)

# Criticality
- How critical is this PR on a 1-10 scale? 4


# Steps to Test

**Note: you can use this branch to test #28, as it includes all of those changes.**

1. Deploy `SITES-482-js-injector` of `acsf-cardinald7` on `dev` or `test`
2. Use test steps from #28 
3. Copy `ds_TELOS`
4. Check that the rule at https://telos-test.cardinalsites.stanford.edu/#overlay=admin/config/development/js-injector/list/GSE%2520Header/edit has `sites/g/files/...` in the rule
5. Check that the file at https://telos-test.cardinalsites.stanford.edu/sites/g/files/sbiybj891/f/js_injector/js_injector_2.js?p8ha76 does as well

# Affected Projects or Products
- Sites 2.0

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/SITES-482
- Other PRs: https://github.com/SU-SWS/acsf-cardinald7/pull/43
- https://github.com/SU-SWS/ansible-playbooks/pull/28
- Anyone who should be notified? ( @sherakama )

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
